EvotoApi/Areas/Management/Connections/RegistrarConnection.cs

```diff
@@ -15,18 +20,38 @@ public class RegistrarConnection
         private const string RegistrarUrl = "http://api.evoto.tech";
 #endif
 
-        private const string ApiBase = "management";
-
-        public static async Task<bool> CreateBlockchain(ManaVote model)
+        private static async Task<IRestResponse> MakeApiRequest(RestRequest req)
         {
             var client = new RestClient(RegistrarUrl);
+            req.AddQueryParameter("key", ConfigurationManager.AppSettings.Get("ApiKeys"));
```

Comment:
Might consider checking this exists first, to save a null reference exception or maybe something else more hard to find